src/sample/Game.java

```diff
@@ -29,7 +30,7 @@ void InitializeGame(View gameView)
         {
             this.bot.RandomizeLocation(gameView.gridSize);
         }
-        gameView.adjustBotAndTreasureLocations(this.bot, this.treasure);
+        // gameView.redrawGrid(this.bot, this.treasure);
```

Comment:
If we don't need this, you can remove it.

Reply:
fairsies enough